src/serializer/functions.js

```diff
@@ -345,16 +338,15 @@ export class Functions {
       originalLocation: BabelNodeSourceLocation | void | null
     ) => {
       let firstLocationString = originalLocation ? `${stringOfLocation(originalLocation)}` : "";
+      let secondLocationString = `${stringOfLocation(location)}`;
       let propString = key ? ` "${key}"` : "";
       let objectString = object ? ` on object "${object}" ` : "";
       if (!objectString && key) objectString = " on <unnamed object> ";
       let error = new CompilerDiagnostic(
-        `Write to property${propString}${objectString}at optimized function ${f1name}[${firstLocationString}] conflicts with access in function ${f2name}[${
-          location.start.line
-        }:${location.start.column}]`,
+        `Write to property${propString}${objectString}at optimized function ${f1name}[${firstLocationString}] conflicts with access in function ${f2name}[${secondLocationString}]`,
         location,
         "PP1003",
-        "FatalError"
+        "RecoverableError"
```

Comment:
I think the issue with this one is that it can end up generating completely invalid code. Not just wrong assumptions. So ignoring it might make it hard to understand a bug. Maybe worth it though.

Reply:
The definition of `RecoverableError` that Herman and I arrived at is: "Prepack might produce code that deviates in behavior from the original code, if the original code is not well behaved."

I think that matches here.